src/contracts/BatchRequests.sol

```diff
@@ -34,8 +34,10 @@ contract BatchRequests is Ownable {
         uint256 contractsLength = contracts.length;
         Batch[] memory batch = new Batch[](contractsLength);
         for (uint256 i; i < contractsLength; ) {
-            bool canBatch = IStaking(contracts[i]).canBatchTransactions();
-            batch[i] = Batch(contracts[i], canBatch);
+            if (contracts[i] != address(0)) {
```

Comment:
storing contracts[i] in a local var will save some gas here as it will avoid the sloads
```
address contract = contracts[i]
...
```